include/blazesym.h

```diff
@@ -114,6 +114,26 @@ typedef enum blazesym_sym_type {
  */
 typedef struct blazesym blazesym;
 
+/**
+ * A normalizer for addresses.
+ *
+ * Address normalization is the process of taking virtual absolute
+ * addresses as they are seen by, say, a process (which include
+ * relocation and process specific layout randomizations, among other
+ * things) and converting them to "normalized" virtual addresses as
+ * they are present in, say, an ELF binary or a DWARF debug info file,
+ * and one would be able to see them using tools such as readelf(1).
+ */
+typedef struct blaze_normalizer blaze_normalizer;
+
+/**
+ * An address normalizer.
+ *
+ * It is returned by [`blaze_normalizer_new()`] and should be freed by
+ * [`blaze_normalizer_free()`].
+ */
+typedef struct blaze_normalizer blaze_normalizer;
```

Comment:
duplicate typedefs?

Reply:
I massaged the Rust code a bit to prevent that from happening. Also added a test compiling a C program.